WinUIGallery/ControlPages/DesignGuidance/AccessibilityScreenReaderPage.xaml

```diff
@@ -28,50 +25,47 @@
             <Setter Property="FontSize" Value="16" />
             <Setter Property="FontWeight" Value="Bold" />
         </Style>
-        <Style x:Key="SampleExplanationTextStyle" TargetType="TextBlock">
-            <Setter Property="FontStyle" Value="Italic" />
-        </Style>
     </Page.Resources>
 
-    <StackPanel
-        MaxWidth="900"
-        HorizontalAlignment="Left"
-        Spacing="10">
-        <RichTextBlock>
-            <Paragraph>
-                Accessibility is about building experiences that make your Windows application usable by people of
-                all abilities. For more information about designing accessible apps:
-                <Hyperlink NavigateUri="https://learn.microsoft.com/windows/apps/design/accessibility/accessibility-overview">Accessibility overview</Hyperlink> .
-                <LineBreak />
-                <LineBreak /> Screen readers, such as
-                <Hyperlink NavigateUri="https://support.microsoft.com/windows/complete-guide-to-narrator-e4397a0d-ef4f-b386-d8ae-c172f109bdb1">Narrator</Hyperlink> , convert text into spoken words to help blind or low vision users.
-                Screen readers use the UI Automation (UIA) names of each control to report their name, role and content.
-                <LineBreak />
-                <LineBreak /> If the control's content is a string, an accessible name is automatically determined from the visible
-                text. However, elements such as images or input fields need to have a custom accessible name. Here
-                are a few examples of how to add UIA names:
-            </Paragraph>
-        </RichTextBlock>
+    <ScrollViewer>
```

Comment:
Scrollbar now overlaps content: 
![Screenshot of Accessibility page](https://user-images.githubusercontent.com/16122379/217065805-c197d859-bba7-4ed7-86ca-9377196da032.png)

Reply:
Same applies for all other a11y pages.